Jellyfin.Server.Implementations/Trickplay/TrickplayManager.cs

```diff
@@ -121,6 +121,13 @@ public async Task RefreshTrickplayDataAsync(Video video, bool replace, Cancellat
                     return;
                 }
 
+                var mediaPath = mediaSource.Path;
+                if (!File.Exists(mediaPath))
+                {
+                    _logger.LogWarning("Media not found: {Path}", mediaPath);
```

Comment:
Please add the item id too

Reply:
Id of the mediasource or the actual video ?

Reply:
Both I guess. Logs should help with troubleshooting so I'd rather have a bit too much info in the log line.